src/librustdoc/clean/inline.rs

```diff
@@ -295,21 +294,24 @@ fn merge_attrs(
     parent_module: Option<DefId>,
     old_attrs: Attrs<'_>,
     new_attrs: Option<Attrs<'_>>,
-) -> clean::Attributes {
+) -> (clean::Attributes, Option<Arc<clean::cfg::Cfg>>) {
     // NOTE: If we have additional attributes (from a re-export),
     // always insert them first. This ensure that re-export
     // doc comments show up before the original doc comments
     // when we render them.
     if let Some(inner) = new_attrs {
-        if let Some(new_id) = parent_module {
-            Attributes::from_ast(old_attrs, Some((inner, new_id)))
-        } else {
-            let mut both = inner.to_vec();
-            both.extend_from_slice(old_attrs);
-            both.clean(cx)
-        }
+        let mut both = inner.to_vec();
+        both.extend_from_slice(old_attrs);
+        (
+            if let Some(new_id) = parent_module {
+                Attributes::from_ast(old_attrs, Some((inner, new_id)))
+            } else {
+                both.clean(cx)
+            },
```

Comment:
It was like this before, but I think it would be more clear to use `from_ast` consistently:
```suggestion
            if let Some(new_id) = parent_module {
                Attributes::from_ast(old_attrs, Some((inner, new_id)))
            } else {
                Attributes::from_ast(both, None)
            },
```

Reply:
Indeed it's better this way